src/main/scala/tech/cryptonomic/conseil/routes/Data.scala

```diff
@@ -8,18 +8,26 @@ import tech.cryptonomic.conseil.config.ServerConfiguration
 import tech.cryptonomic.conseil.generic.chain.DataPlatform
 import tech.cryptonomic.conseil.generic.chain.DataTypes.QueryResponseWithOutput
 import tech.cryptonomic.conseil.metadata.{EntityPath, MetadataService, NetworkPath, PlatformPath}
-import tech.cryptonomic.conseil.tezos.ApiOperations
+import tech.cryptonomic.conseil.tezos.{ApiOperations, TezosDatastore}
 import tech.cryptonomic.conseil.tezos.TezosTypes.{AccountId, BlockHash}
 import tech.cryptonomic.conseil.util.ConfigUtil
 
+import cats.implicits._
+
 import scala.concurrent.{ExecutionContext, Future}
 
 /** Companion object providing apply implementation */
 object Data {
-  def apply(config: PlatformsConfiguration, metadataService: MetadataService, server: ServerConfiguration)(
+  def apply(
+      config: PlatformsConfiguration,
+      metadataService: MetadataService,
+      server: ServerConfiguration,
+      apis: ApiOperations,
+      datastore: TezosDatastore
+  )(
       implicit ec: ExecutionContext
   ): Data =
-    new Data(config, DataPlatform(server.maxQueryResultSize), metadataService)
+    new Data(config, DataPlatform(server.maxQueryResultSize, apis), metadataService, apis, datastore)
```

Comment:
What's the purpose of `DataPlatform`? I see that you pass `ApiOperations` twice and I wonder if it's not mixing the levels of abstraction.

Reply:
It was supposed to be a middle level, to add possibility of using multiple platforms with one Conseil instance. But at the moment we don't support multiple platforms.

Reply:
I tried to refactor to make it work, we could eventually revisit how those dependencies work and make more sensible choices